Upstream: No, Alpine specific
Reason: We don't have that file, so we can't mount it into the sandbox
diff --git a/libgnome-desktop/gnome-desktop-thumbnail-script.c b/libgnome-desktop/gnome-desktop-thumbnail-script.c
index 0aa2644..9260f0c 100644
--- a/libgnome-desktop/gnome-desktop-thumbnail-script.c
+++ b/libgnome-desktop/gnome-desktop-thumbnail-script.c
@@ -540,7 +540,6 @@ add_bwrap (GPtrArray   *array,
   add_args (array,
 	    "bwrap",
 	    "--ro-bind", "/usr", "/usr",
-	    "--ro-bind", "/etc/ld.so.cache", "/etc/ld.so.cache",
 	    NULL);
 
   /* These directories might be symlinks into /usr/... */
